src/wp-login.php

```diff
@@ -791,7 +791,7 @@ function wp_login_viewport_meta() {
 			$secure = false;
 		}
 
-		setcookie( 'wp-postpass_' . COOKIEHASH, $hasher->HashPassword( wp_unslash( $_POST['post_password'] ) ), $expire, COOKIEPATH, COOKIE_DOMAIN, $secure );
+		setcookie( 'wp-postpass_' . COOKIEHASH, $hasher->HashPassword( wp_unslash( $_POST['post_password'] ) ), $expire, COOKIEPATH, COOKIE_DOMAIN, $secure, true );
```

Comment:
I support setting the test cookie and `wp_lang` cookie to HttpOnly, but I don't think it's safe to do it for the post password cookie. There's too high an opportunity for it to break either JavaScript that reads the value of the cookie, or Ajax handlers in PHP that read the value of the cookie, both of which would no longer be able to do so.

Let's revert this change but keep the other changes.